content/blog/20210712_fluentd-v1.13.2-has-been-released.md

```diff
@@ -0,0 +1,69 @@
+# Fluentd v1.13.2 has been released
+
+Hi users!
+
+We have released v1.13.2. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#v1132).
+
+This release is a maintenance release of v1.13 series.
+We recommend to upgrade Fluentd because it contains fixes about crash bugs.
+
+### Fixed a crash bug during startup phase
+
+In this release, a crash bug was fixed during outputting log at the early stage when parsing invalid config file.
+
+This was a bug since v1.13.0. If you use invalid '@' prefix parameter, remove it as a workaround.
+
+### `in_tail`: Fixed a bug that remaining lines will be discarded
+
+When rotation is occurred with the throttling feature is enabled,
+there is a case that remaining lines will be discarded.
+
+### Added support to customize configuration of oj options
+
+In this release, it is able to customize OJ options via environment variables.
+
+Use the following environment variables to customize behavior of OJ.
+
+* `FLUENT_OJ_OPTION_BIGDECIMAL_LOAD` (float by default)
+* `FLUENT_OJ_OPTION_MAX_NESTING`
+* `FLUENT_OJ_OPTION_MODE` (compat by default)
+* `FLUENT_OJ_OPTION_USE_TO_JSON` (true by default)
+
+See [Oj Options](https://github.com/ohler55/oj/blob/develop/pages/Options.md) about details of oj options.
+
+### Generating template of storage plugin was supported
+
+Since v1.13.2, `fluent-plugin-generate` command has been supported to generate template files for storage plugin.
+
+```
+$ fluent-plugin-generate storage foo
+License: Apache-2.0
+        create Gemfile
+        create README.md
+        create Rakefile
+        create fluent-plugin-foo.gemspec
+        create lib/fluent/plugin/storage_foo.rb
+        create test/helper.rb
+        create test/plugin/test_storage_foo.rb
+Initialized empty Git repository in /work/fluentd/fluentd/fluentd/fluent-plugin-foo/.git/
+```
+
+### Upgrade attention for TD Agent users
+
+If you use Fluentd v1.12 or later, we recommend to use at least v1.12.4.
+If you use Fluentd v1.13, we recommend to use at least v1.13.2.
```

Comment:
For in_tail's bug, 1.13.0 is enough to fix.

Reply:
Fixed and added explanation a bit.